subsys/fs/Kconfig.littlefs

```diff
@@ -6,8 +6,7 @@
 config FILE_SYSTEM_LITTLEFS
 	bool "LittleFS support"
 	depends on FILE_SYSTEM
-	depends on FLASH_MAP
-	depends on FLASH_PAGE_LAYOUT
+	depends on (FLASH_MAP && FLASH_PAGE_LAYOUT) || MMC_STACK
```

Comment:
Instead of doing this, the approach to Kconfig from https://github.com/zephyrproject-rtos/zephyr/pull/41972/files#diff-b12838b5f8feaf474fa81aeab169e1a7ac9765d34c81f76845f658425bd71d79R17 should be considered, where there are two separate options to allow FLASH_MAP based and DISK_ACCESS based LittleFS independently, as user may use both of them at runtime.

Reply:
done